frontend/src/components/Util/TaskEditors/FloatingTaskEditor.module.css

```diff
@@ -4,7 +4,7 @@
   z-index: 8;
   box-shadow: 0 5px 12px 0 rgba(0, 0, 0, 0.3);
   transition: width 0.1s linear;
-  max-height: 400px;
+  max-height: 200px;
```

Comment:
I'm surprised that decreasing the height can solve the problem, instead of increasing it.

Reply:
Yeah, the decreased height makes the editor overflow if there are enough subtasks for it to go below the viewport, and then the user can use the scrollbar to change the due date / save.